release.sh

```diff
@@ -0,0 +1,79 @@
+#! /bin/bash
+set -euo pipefail
+#
+# Source: https://gist.github.com/philbuchanan/8188898
+# License: GNU GPL 2.1
+# Script to deploy from Github to WordPress.org Plugin Repository
+# A modification of Dean Clatworthy's deploy script as found here: https://github.com/deanc/wordpress-plugin-git-svn
+# The difference is that this script lives in the plugin's git repo & doesn't require an existing SVN repo.
+# Slightly adapted
+
+SVNUSER=razorpay
+PLUGINSLUG='razorpay-payment-button'
```

Comment:
Change the slug name here.

Reply:
slug name will get from wordpress.org

Reply:
Wordpress.org will pick the name from the plugin header. So you can name the slug according to that. @neera11

Reply:
changed